samples/DotnetCore/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/sdk:9.0-alpine AS build
+
+ARG client_version
```

Comment:
`docker build --tag testapp:version . --build-arg client_version=version`

ie. if you want a specific version of the .NET SDK (5.5.3 for instance)